library/.eslintrc

```diff
@@ -204,5 +245,9 @@
     "jsx-a11y/role-has-required-aria-props": "warn",
     "jsx-a11y/role-supports-aria-props": "warn",
     "jsx-a11y/scope": "warn"
+  },
+
+  "settings": {
+    "import/ignore": [/\.(css|png|jpe?g|gif)$/]
```

Comment:
I don't think this is wise, this list will be potentially endless. Doesn't `import/extensions` with a value of `['.js']` just do the trick?

Reply:
Perhaps, I'll dive into it.